Sources/Plasma/FeatureLib/pfLocalizationMgr/CMakeLists.txt

```diff
@@ -2,14 +2,27 @@ include_directories(../../CoreLib)
 include_directories(../../NucleusLib/inc)
 include_directories(../../PubUtilLib)
 
-include_directories(${EXPAT_INCLUDE_DIR})
+# Skip CMake's built in FindExpat module and use libexpat's CMake config, if possible,
+# due to library rename in libexpat 2.2.8 and higher.
+find_package(EXPAT CONFIG QUIET)
+if(NOT TARGET expat::libexpat)
+    message(DEBUG "libexpat CMake Config not found, using FindExpat module.")
+    find_package(EXPAT REQUIRED)
+    add_library(expat::libexpat STATIC IMPORTED)
+    set_target_properties(expat::libexpat PROPERTIES
+                          INTERFACE_INCLUDE_DIRECTORIES ${EXPAT_INCLUDE_DIR}
+                          IMPORTED_LOCATION ${EXPAT_LIBRARY}
+    )
+endif()
+
+set_package_properties(EXPAT PROPERTIES
+    URL "https://github.com/libexpat/libexpat"
+    DESCRIPTION "Expat XML Parser for C"
+    TYPE REQUIRED
+)
 
 if(WIN32)
     add_definitions(-DWIN32)
-
-    if(EXPAT_IS_STATIC)
-        add_definitions(-DXML_STATIC)
```

Comment:
It looks like the "fix" for this is vcpkg-specific, so if we want to support non-vcpkg versions of expat, we should keep this :(

Reply:
😢